types/codemirror/index.d.ts

```diff
@@ -1222,6 +1222,153 @@ declare namespace CodeMirror {
      */
     function overlayMode<T, S>(base: Mode<T>, overlay: Mode<S>, combine?: boolean): Mode<any>;
 
+    interface CommandActions {
+        /** Select the whole content of the editor. */
+        selectAll(): void;
```

Comment:
Shouldn't this have `cm: CodeMirror.Editor` argument too?

Reply:
My bad... Fixed.